servers/shares-api/schema.graphql

```diff
@@ -1,3 +1,72 @@
+extend schema
+  @link(url: "https://specs.apollo.dev/link/v1.0")
+  @link(
+    url: "https://specs.apollo.dev/federation/v2.3"
+    import: ["@key", "@tag"]
+  )
+  # The link directive is required to federate @constraint
+  # It doesn't actually have to be a real spec, but it would be good
+  # to write one and replace this.
+  @link(url: "https://myspecs.dev/myDirective/v1.0", import: ["@constraint"])
+  @composeDirective(name: "@constraint")
+
+scalar Url
+scalar Markdown
+scalar ISOString
+
 type PocketShare {
-  id: ID!
-}
\ No newline at end of file
+  slug: ID!
```

Comment:
This also contains `preview: ItemSummary` (federated via parser-graphql-wrapper)

Reply:
I'm gonna rename it to preview in my pr. This name makes more sense.